packages/mgt-chat/src/components/MgtChat/MgtChat.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import { FluentThemeProvider, MessageThread } from '@azure/communication-react';
+import { useGraphChatClient } from '../../statefulClient/useGraphChatClient';
+import { registerIcons } from '@fluentui/react';
+import { DEFAULT_COMPONENT_ICONS } from '@azure/communication-react';
```

Comment:
Is there any reason for having L5 import outside L2 imports?

Reply:
Just me not spotting that